lib/delayed/backend/actions.rb

```diff
@@ -31,7 +31,11 @@ def db_time_now
 
         def find_available(worker_name, limit = 5, max_run_time = Worker.max_run_time)
           Delayed::Worker.available_priorities.each do |priority|
-            message = ironmq.queue(queue_name(priority)).get
+            begin
+              message = ironmq.queue(queue_name(priority)).get
```

Comment:
message defined within begin/rescue block and could not be available outside (in case of exception e.g.)